pfcwd/main.py

```diff
@@ -110,6 +111,10 @@ def __init__(
     ):
         self.db = None
         self.config_db = None
+
+        # Load database config files
+        load_db_config()
+
```

Comment:
This did not help resolve the issue seen on multi-asic VS.
Adding at the cli entry point worked:
```
# Main entrypoint
@click.group()
def cli():
    """ SONiC PFC Watchdog """
    load_db_config()
```

Reply:
How about moving to `__main__` section? It should be the entry point of this script.

Reply:
Adding it to __main__ did not help:
```
if __name__ == '__main__':
    load_db_config()
    cli = get_pfcwd_clis()
    cli()
```

Reply:
pfcwd start_default invokes: /usr/local/bin/pfcwd ; which invokes cli() in pfcwd.main.
```
admin@vlab-08:~$ vi /usr/local/bin/pfcwd 
#!/usr/bin/python3
# -*- coding: utf-8 -*-
import re
import sys
from pfcwd.main import cli
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(cli()) 
```

Reply:
This is cool finding. `__main__` section is not the entry point of this script.

Reply:
__main__ also invokes cli(), so having this change in cli() will ensure that invoking main also works.

Reply:
Fixed, move load_db_config to cli()